templates/gov/index.html

```diff
@@ -22,14 +22,51 @@ <h1>Secure and certified open source for Public Sector</h1>
           height="200",
           width="350",
           hi_def=True,
-          attrs={"class": ""},
           loading="auto",
         ) | safe
       }}
     </div>
   </div>
 </section>
-
+<section class="p-strip--square-suru">
```

Comment:
This is duplicated between the two pages, so I think it should move to a partial and just be included on both pages.